src/com/cft/contactmerge/io/SplitStringUtilities.java

```diff
@@ -0,0 +1,17 @@
+package com.cft.contactmerge.io;
+
+import java.util.*;
+
+public class SplitStringUtilities {
+    public static Collection<String> splitTsvString(String input)
+    {
+        // TODO: Add code to return appropriate results
+        return null;
+    }
+
+    public static Collection<String> splitCsvString(String input)
+    {
```

Comment:
I think java has a library built for parsing csv files. It may be worth looking into -> https://commons.apache.org/proper/commons-csv/apidocs/org/apache/commons/csv/CSVParser.html

Reply:
Definitely. I'll leave that up to the person developing the code.